community/libev/build

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh -e
+
+# export V=1 for verbose output when building
+[ "${V:-0}" -eq 1 ] > /dev/null 2>&1 && set -x || :
+
+PATCHES="${PATCHES:-}"
+
+PATCHES="
+libev-4.25-pc.patch
+${PATCHES}
+"
+
+if [ -n "${PATCHES}" ]
+then
+	for patch in ${PATCHES}
+	do
+		if [ -z "${patch}" ] || [ ! -e "${patch}" ]
+		then
+			continue
+		fi
+		patch -p1 < "${patch}"
+	done
+fi
```

Comment:
Everything above this (minus the `shebang` should be removed). Patches should be done with a for loop if `-p` matches for all of them or one per line if it doesn't.